api/net/ip4/ip4.hpp

```diff
@@ -306,7 +306,32 @@ namespace net {
     PMTU minimum_MTU() const noexcept
     { return (PMTU) PMTU_plateau::ONE; }
 
+    ip4::Addr ip4_addr() const noexcept
```

Comment:
Uncertain of these functions need to contain the `ip4_` prefix due to being a member in IP4. What do you think?